book/content/open-source-comms/comms-channels.md

```diff
@@ -0,0 +1,34 @@
+# Communication Channels
+
+The channel(s) through which you choose to communicate with your community will have a big impact on the experience and sense of belonging for your community members.
+The choice of channels will depend on the specific requirements of your project, such as keeping the community informed of any known bugs or upcoming features, assessing the current needs from the community members, inviting comments on the management plans, facilitating the overall progress of the project, as well as having somewhere to field questions and feedback.
+
+When setting up your communication channels, there are some important things to consider such as the following:
+
+* How easy is it for the community to find these channels and subscribe/unsubscribe to them?
+* How easy is it for members of the community to find and join specific conversations they are interested in?
+* Will it be a "noisy" channel (for example, lots of notifications)?
+* Can users control or filter what kind of information they receive?
+
+There are three commonly used channels: mailing lists (such as [Topicbox](https://www.topicbox.com/), or [Google groups](https://support.google.com/mail/thread/14635045?hl=en)), community forums (such as [Vanilla Forum](https://vanillaforums.com/en/software/) or [Discourse](https://www.discourse.org/)) and Internet Relay Chat (IRC) (such as [Gitter](https://gitter.im/) or [Slack](https://app.slack.com/signin)). 
```

Comment:
IRC is like Slack or Gitter (a kind of chat). Together they are "chats" I'd say.

Reply:
Ahha. I assumed that it's a common umbrella term. Thanks for pointing that. 😇